custom_components/opnsense/sensor.py

```
@@ -722,24 +715,17 @@
-    @property
-    def extra_state_attributes(self) -> Mapping[str, Any]:
-        temp: Mapping[str, Any] = self._opnsense_get_temp()
-        attributes: Mapping[str, Any] = {}
-        for attr in ["device_id"]:
-            attributes[attr] = temp.get(attr, None)
-        return attributes
-
-    @property
-    def native_value(self):
+    @callback
+    def _handle_coordinator_update(self) -> None:
```

Ruff

Missing type annotation for `self` in method

(ANN101)